pkg/virt-operator/webhooks/kubevirt-update-admitter_test.go

```diff
@@ -117,12 +117,12 @@ var _ = Describe("Validating KubeVirtUpdate Admitter", func() {
 		},
 			Entry("with unspecified minTLSVersion but non empty ciphers",
 				&v1.TLSConfiguration{Ciphers: []string{tls.CipherSuiteName(tls.TLS_AES_256_GCM_SHA384)}},
-				"You cannot specify ciphers when spec.configuration.tlsConfiguration.minTLSVersion is empty or VersionTLS13",
+				"You cannot specify non-compliant ciphers when spec.configuration.tlsConfiguration.minTLSVersion is empty or VersionTLS13",
 				-1,
 			),
 			Entry("with specified ciphers and minTLSVersion = 1.3",
 				&v1.TLSConfiguration{Ciphers: []string{tls.CipherSuiteName(tls.TLS_AES_256_GCM_SHA384)}, MinTLSVersion: v1.VersionTLS13},
-				"You cannot specify ciphers when spec.configuration.tlsConfiguration.minTLSVersion is empty or VersionTLS13",
+				"You cannot specify non-compliant ciphers when spec.configuration.tlsConfiguration.minTLSVersion is empty or VersionTLS13",
```

Comment:
sorry for jumping in between , is there a way to identify and print non compliant cipher.

Reply:
Thanks for the question. digging to check the name made it clear this PR is not needed.